simcon_project/simcon_project/urls.py

```diff
@@ -15,7 +15,9 @@
 """
 from django.contrib import admin
 from django.urls import path
-from users.views import RedirectFromLogin, StudentView, ResearcherView, ViewResponse, UpdateOverallResponseFeedback, UpdateNodeResponseFeedback, ResearcherSettingsView, TemplateStartView, ResearcherUserView, StudentSettingsView,StudentUserView
```

Comment:
We keep changing this back and forth between PRs so we should stick with one or the other. I personally like this better since it's more readable

Reply:
It's not very readable for me since I use a portrait-style monitor to code often but I'm game with either I guess. I think we're getting close to never having to look at that line again so no biggie.

Reply:
Oh i might'vetotally misread what you wrote. Are you saying you like it multi-line? or all on one line?